Source/WebCore/platform/cf/RunLoopObserverCF.cpp

```diff
@@ -36,6 +36,8 @@ static constexpr CFIndex cfRunLoopOrder(RunLoopObserver::WellKnownOrder order)
     switch (order) {
     case RunLoopObserver::WellKnownOrder::GraphicsCommit:
         return coreAnimationCommit;
+    case RunLoopObserver::WellKnownOrder::PostGraphicsCommit:
+        return coreAnimationCommit + 1;
```

Comment:
`coreAnimationCommit + 1` is already used for `InspectorFrameEnd`. Should we ensure the order numbers we return are unique?

Reply:
`WellKnownOrder::InspectorFrameEnd` is unused and should be removed, so this index clash doesn't matter.